### ISSUE TITLE
Configure unicorn as backup production server for pegasus

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#473]
Anticipated on ~~Aug22~~ Aug19, we'll be bringing pegasus down for about a day for rebuilding. It would be nice to have a backup server ready-to-go, even not fully functional, during this gap, as well as available as a fallback server if pegasus has problems, so that the image.a11y.mcgill.ca website remains live, and the IMAGE browser extension continues to function.

- [ ] let preprocessor(s) fall back to public OSM servers if pegasus is unresponsive (#472)
- [ ] have a process for Nick targeting pegasus.cim.mcgill.ca to a different IP, or create a way for us to do this ourselves (MAC spoof?)
- [ ] implement a way to sync the resources/models directories since these currently only exist on pegasus.
- [ ] once previous is done, confirm that unicorn accesses these resources locally rather than hitting pegasus. (@Cybernide we'd have to update website links to be local, I think?)

@JRegimbal @Cybernide @rohanakut @jaydeepsingh25 can you think of other things that would need to be done to allow unicorn to be a temporary standalone server for both the website and the production back-end, assuming pegasus is completely inaccessible?

**Comment by JRegimbal** [comment]
RE: point 1, what does this mean if the OSM preprocessor is not actually merged? For point 3/4, we could probably set up an rsync cron job and update those files at midnight. Then we could update the website files to use relative rather than absolute links.

**Comment by jeffbl** [comment]
If not merged, we should still do this since I'm sure it won't be the last time we have to do downtime or have a high-stakes event where we'd like to have a backup server... rsync could work, but once we start down that path, maybe the synologies should serve a the main location, with however many servers syncing from there? At that point, it almost sounds like using [syncthing](https://syncthing.net/) might be a good idea...

**Comment by jeffbl** [comment]
As evidenced yesterday, unicorn is now capable of functioning as a backup server when needed. We just need pegasus.cim.mcgill.ca pointed to unicorn.cim.mcgill.ca.. Few notes:
1. website seemed to work fine, but of course the `resources` directory needs to be compatible with the version of the website checked out on unicorn.
2. MOTD comes in handy to let people know they are on a backup server, so to expect slower/different behaviour.
3. the dockewr-compose.yml for unicorn is very different than that for pegasus. For extended running, we'd want to use something closer to the pegasus docker-compose.
